### PR TITLE
Frontend websocket

### PR DESCRIPTION
**Pull request by oskar-dah**
This addresses issues number #28 & #32. 
Field to show analysis status:
image

.env file to handle IP addresses to backend.

### CODE REVIEW

**Comment by oskar-dah** [comment]
> Is this supposed to work with backend right now? The upload functionality does not seem to work. Do not know if the LUDD server is broken of the code is broken. I think that the frontend can not communicate with the backend.

We have updated the docker files so it should work now. You may need to view the page in incognito mode in your browser to see you latest changes after building and running it with docker.

File: frontend/frontend.sh [hunk 1]
```diff
@@ -4,12 +4,13 @@ COMPOSE_FILE="docker-compose.yaml"
 
 build_frontend() {
     echo "Building frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE build frontend
+    docker-compose  -f $COMPOSE_FILE build frontend --no-cache
```
Comment (Lundsak) [inline frontend/frontend.sh:7]: --no-cache has to be before after build and before frontend to work
docker-compose  -f $COMPOSE_FILE build --no-cache frontend

File: frontend/frontend.sh [hunk 2]
```diff
@@ -4,12 +4,13 @@ COMPOSE_FILE="docker-compose.yaml"
...
 }
 
 run_frontend() {
     echo "Running frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE --compatibility up -d frontend
+    docker-compose down
+    docker-compose -f $COMPOSE_FILE --compatibility up -d frontend --build
```
Comment (Lundsak) [inline frontend/frontend.sh:13]: docker-compose -f $COMPOSE_FILE --compatibility up -d  --build frontend

Comment (oskar-dah) [reply frontend/frontend.sh:13]: Yes, we will change the order of the command. Unclear why this is needed but we can't find any other way of making it work.

File: frontend/.env
Comment (IsakLundstrom) [inline frontend/.env:1]: Kanske bara borde vara en variabel? typ `REACT_APP_BACKEND_ADRESS=localhost:8080`

**Review by ClaudeHallard — APPROVED** [review]

File: frontend/frontend.sh [hunk 3]
```diff
@@ -4,12 +4,13 @@ COMPOSE_FILE="docker-compose.yaml"
...
     echo "Building frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE build frontend
+    docker-compose  -f $COMPOSE_FILE build frontend 
 }
 
 run_frontend() {
     echo "Running frontend Docker service..."
-    docker-compose -f $COMPOSE_FILE --compatibility up -d frontend
+    docker-compose down
+    docker-compose -f $COMPOSE_FILE --compatibility up -d --build frontend
```
Comment (IsakLundstrom) [inline frontend/frontend.sh:13]: Alltså jag tänker att det inte borde vara --build här, men om det måste vara det så måste det väl det, men Lundmark behövde det i alla fall inte

**Pull request merged**